### PR TITLE
Adds navigation.

### PR DESCRIPTION
**Pull request by VeronikaYastr**
Adds navigation with simple empty screens.

### CODE REVIEW

File: HeatApp/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const MainTabs = createBottomTabNavigator(
+  {
+    Favourite: FavouritesScreen,
+    Menu: MenuScreen,
+    Settings: UserSettingsScreen
+  },
+  {
+    initialRouteName : 'Menu',
+    tabBarOptions: {
+      activeTintColor: "#308312"
```
Comment (daksenik) [inline HeatApp/App.js]: Let's move color definition to some file with constants, so we can be consistent about our color scheme.

File: HeatApp/components/screens/StartSettingsScreen.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  startBtn: {
+    width: WIDTH - 55,
+    height: 45,
+    borderRadius: 25,
+    backgroundColor: "rgba(21, 211, 102, 0.8)",
```
Comment (daksenik) [inline HeatApp/components/screens/StartSettingsScreen.js]: Same for color and all other colors in this PR. It's better to store them in a single file as constants. Background color can be hex, can't it?

**Review by daksenik — CHANGES_REQUESTED** [review]

File: HeatApp/App.js (hunk 1 above)
Comment (VeronikaYastr) [reply HeatApp/App.js]: No problem)

File: HeatApp/components/screens/StartSettingsScreen.js (hunk 2 above)
Comment (VeronikaYastr) [reply HeatApp/components/screens/StartSettingsScreen.js]: Yes, of course it can be) It's just example))

**Comment by VeronikaYastr** [comment]
I think we should use StackNavigator inside of TabsNavigator for cases when we, for example, tap on card with dish and go to it's description. StackNavigator will hide bottom tabs and add button "back" at the top of the screen. Do you think the same or you think we should leave tabs? @daksenik

**Comment by daksenik** [comment]
> I think we should use StackNavigator inside of TabsNavigator for cases when we, for example, tap on card with dish and go to it's description. StackNavigator will hide bottom tabs and add button "back" at the top of the screen. Do you think the same or you think we should leave tabs? @daksenik

StackNavigator sounds good to me. But let's merge this PR first, so we're unblocked on moving forward, since the StackNavigator seems to be a single-component change.

**Pull request merged**